dist/index.js

```diff
@@ -8,6 +8,11 @@ module.exports = scan
 module.exports.cli = scanCLI
 module.exports.customScanner = scanUsingCustomSonarQubeScanner
 
+let foo
+if (foo == '') {
+  
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the useless trailing whitespaces at the end of this line. ([javascript:TrailingWhitespace](https://sonarcloud.io/organizations/default/rules#rule_key=javascript%3ATrailingWhitespace))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonarqube-scanner&pullRequest=38&issues=AWLNyHlbjr7wspRoiHiR&open=AWLNyHlbjr7wspRoiHiR)

[ISSUE_KEY]: AWLNyHlbjr7wspRoiHiR